geoips/plugins/modules/output_checkers/image.py

```diff
@@ -22,6 +22,59 @@
 name = "image"
 
 
+def get_test_files():
+    """Return a series of compare vs output image paths for testing purposes."""
+    from PIL import Image
+    import numpy as np
+    from importlib.resources import files
+    from os import makedirs
+    from os.path import exists
+
+    savedir = str(files("geoips") / "../../test_data/test_images/pytest") + "/"
```

Comment:
This resolved
https://github.com/NRLMMD-GEOIPS/geoips/pull/379#discussion_r1366934180